example/lib/main.dart

```diff
@@ -9,9 +9,24 @@ class ZetaExample extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Zeta(
-      builder: (context, theme, colors) {
-        return MaterialApp.router(theme: theme, routerConfig: router);
+    return ZetaProvider(
+      builder: (context, themeData, themeMode) {
+        final dark = themeData.colorsDark.toScheme();
+        final light = themeData.colorsLight.toScheme();
+        return MaterialApp.router(
+          routerConfig: router,
+          themeMode: themeMode,
+          theme: ThemeData(
```

Comment:
Can we create a helper to build these theme objects?

Reply:
We could introduce a helper function, but it might not be necessary since we'll have theme builders in the Zds repo. If required, we can transfer those builders to Zeta.